tests/test_picking.py

```diff
@@ -124,6 +124,35 @@ def callback(point):
     assert pl.picked_point is None
 
 
+@pytest.mark.parametrize('left_clicking', [False, True])
+def test_disable_picking(sphere, left_clicking):
+    picked = []
+
+    def callback(point):
+        picked.append(point)
```

Comment:
Are we using `picked` anywhere? If not, can't we make `callback` a no-op?

Reply:
Resolved in 3613f24db71a65292fae88c30cdbee95fd99adeb.